### PR TITLE
[PKG-5138] Update to 0.19.1

### PR DESCRIPTION
**Pull request by ViridianMelody**
tokenizers 0.19.1

**Destination channel:** defaults

### Links

- [PKG-5138]
- [Upstream repository](https://github.com/huggingface/tokenizers)
- [Upstream changelog/diff](https://github.com/huggingface/tokenizers/compare/v0.15.1...v0.19.1)

### Explanation of changes:

- Added upstream tests. Excluded Windows as multiprocess forking isn't supported.

[PKG-5138]: https://anaconda.atlassian.net/browse/PKG-5138?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -49,8 +52,16 @@ test:
     - huggingface_hub
   commands:
     - pip check
+    - cd bindings/python
+    # tests require multiprocessing with fork, which is not available on Windows.
+    - pytest tests --ignore="tests/documentation" -k "not test_continuing_prefix_trainer_mistmatch"  # [not win]
```
Comment (Jrice1317) [inline recipe/meta.yaml:60]: Are there any tests that can be ran for Windows?

**Review by Jrice1317 — APPROVED** [review]
Outside of testing for `win`, LGTM!

File: recipe/meta.yaml (hunk 1 above)
Comment (danpetry) [reply recipe/meta.yaml:60]: add `not multiprocessing` for win?

**Review by danpetry — APPROVED** [review]
based on jaida's comment being addressed

**Comment by danpetry** [comment]
not sure if this is helpful, but pytest's `-k` flag will match substrings of the test names:
```
general:
  -k EXPRESSION         Only run tests which match the given substring
                        expression. An expression is a Python evaluable
                        expression where all names are substring-matched against
                        test names and their parent classes. Example: -k
                        'test_method or test_other' matches all test functions
                        and classes whose name contains 'test_method' or
                        'test_other', while -k 'not test_method' matches those
                        that don't contain 'test_method' in their names. -k 'not
                        test_method and not test_other' will eliminate the
                        matches. Additionally keywords are matched to classes
                        and functions containing extra names in their
                        'extra_keyword_matches' set, as well as functions which
                        have names assigned directly to them. The matching is
                        case-insensitive.
```

**Pull request merged**